src/store/user/actions.tsx

```diff
@@ -0,0 +1,41 @@
+/*--------------------LOGIN--------------------*/
+
+export function loginThunk(data) {
+  return function(dispatch) {
+    console.log(`data after login front end ----------->`, data);
+    if (data) return dispatch(loginSuccess(data));
+  };
+}
```

Comment:
I don't think you need this thunk right? if im not wrong it's a leftover from when you where using REST login. You could just dispatch from the component loginSuccess.
Now, if you are using the pattern of importing actions and then passing them through mapDispatchToProps and have no actual dispatches in the component then maybe you can keep this to be consistent in the code style in the components

Reply:
yes, I actually thought about that. It's just the way I've used it before. No need actually as none of them are reused over several components